src/radical/saga/adaptors/base.py

```diff
@@ -40,7 +40,9 @@ def __init__ (self, adaptor_info, adaptor_options=None) :
         self._logger  = ru.Logger('radical.saga.api')
 
 
-        self._cfg     = ru.Config(module='radical.saga', name=self._name)
+        # we need to expand later once we got env from the remote resource
+        self._cfg     = ru.Config(module='radical.saga', name=self._name,
+                                  expand=False)
```

Comment:
As we are changing the base class, are we changing the default expand behavior for all the adaptors?

Reply:
Yes, indeed, good point.  The `shell_job` adaptor is the only active adaptor in use which uses that env expansion on the target host.  But let me clean this up for the adaptors which use local expansion.

Thanks.